packages/ma-components/local/page-header/page-header.css

```
@@ -0,0 +1,57 @@
+.ma-page-header {
+  background-color: var(--ma-page-header-background-color);
+  container-name: ma-page-header;
+  container-type: inline-size;
+  inline-size: 100%;
+  position: relative;
+  z-index: 0;
+}
+
+.ma-page-header__content {
+  border-block-end-style: solid;
+  border-block-end-width: var(--ma-page-header-border-block-end-width);
+  border-color: var(--ma-page-header-border-color);
+  box-sizing: border-box;
+  display: flex;
+  margin-inline: auto;
+  max-inline-size: var(--ma-page-header-content-max-inline-size);
+  padding-block-end: var(--ma-page-header-content-padding-block-end);
+  padding-block-start: var(--ma-page-header-content-padding-block-start);
+  padding-inline-end: var(--ma-page-header-content-padding-inline-end);
+  padding-inline-start: var(--ma-page-header-content-padding-inline-start);
+}
+
+.ma-page-header__group {
+  display: flex;
+  flex: 1;
+}
+
+.ma-page-header__group--left {
+  justify-content: left;
+}
+
+.ma-page-header__group--right {
+  justify-content: end;
+}
```

Nu is het logical en physical door elkaar. Kan dit misschien consistent logical properties gebruiken, of juist consistent niet? Logo enzo is in right-to-left gespiegeld. Maar misschien is de API wel gebaseerd op slots voor buttons voor rechtshandige mensen 🤷‍♂️
```suggestion
.ma-page-header__group--start {
  justify-content: start;
}

.ma-page-header__group--end {
  justify-content: end;
}
```